rules/windows/persistence_remote_password_reset.toml

```
@@ -34,19 +36,30 @@ type = "eql"
 query = '''
 sequence by winlog.computer_name with maxspan=1m
   [authentication where event.action == "logged-in" and
+
     /* event 4624 need to be logged */
     winlog.logon.type : "Network" and event.outcome == "success" and source.ip != null and
     source.ip != "127.0.0.1" and source.ip != "::1" and
-    not winlog.event_data.TargetUserName : ("svc*", "PIM_*", "_*_", "*-*-*", "*$")] by winlog.event_data.TargetLogonId
+    not startsWith~(winlog.event_data.TargetUserName, "svc") and
+    not startsWith~(winlog.event_data.TargetUserName, "PIM_") and
+    not endsWith~(winlog.event_data.TargetUserName, "$") and
+    not (startsWith(winlog.event_data.TargetUserName, "_") and endsWith(winlog.event_data.TargetUserName, "_")) and
+    not winlog.event_data.TargetUserName regex "[a-zA-Z0-9]+-[a-zA-Z0-9]+-[a-zA-Z0-9]+"] by winlog.event_data.TargetLogonId
    /* event 4724 need to be logged */
   [iam where event.action == "reset-password" and
    (
     /*
-       This rule is very noisy if not scoped to privileged accounts, duplicate the
-       rule and add your own naming convention and accounts of interest here.
+       This rule is very noisy if not scoped to custom privileged accounts.
+       If noisy, duplicate the rule and replace the filters below with your own naming convention and accounts of interest.
      */
-    winlog.event_data.TargetUserName: ("*Admin*", "*super*", "*SVC*", "*DC0*", "*service*", "*DMZ*", "*ADM*") or
-    winlog.event_data.TargetSid : ("S-1-5-21-*-500", "S-1-12-1-*-500")
+    stringContains(winlog.event_data.TargetUserName, "Admin") or
+    stringContains(winlog.event_data.TargetUserName, "super") or
+    stringContains(winlog.event_data.TargetUserName, "SVC") or
+    stringContains(winlog.event_data.TargetUserName, "DC0") or
+    stringContains(winlog.event_data.TargetUserName, "service") or
+    stringContains(winlog.event_data.TargetUserName, "DMZ") or
+    stringContains(winlog.event_data.TargetUserName, "ADM") or
+    winlog.event_data.TargetSid regex~ "S-1-(5-21|12-1)-.*-500"
```

TargetSid is always uppercase + numbers
```suggestion
    winlog.event_data.TargetSid regex "S-1-(5-21|12-1)-.*-500"
```